rabbitmq/assets/monitors/message_unacknowledge_rate_anomaly.json

```
@@ -0,0 +1,33 @@
+{
+  "name": "[RabbitMQ] Messages unacknowledged rate is higher than usual on: {{host.name}}",
+  "type": "query alert",
+  "query": "avg(last_4h):anomalies(avg:rabbitmq.queue.messages_unacknowledged.rate{*} by {rabbitmq_queue,host}, 'agile', 2, direction='above', alert_window='last_15m', interval=60, count_default_zero='true', seasonality='hourly') >= 1",
+  "message": "The rate at which messages are being delivered without receiving acknowledgement is higher than usual. There may be errors or performance issues downstream.\n\nHost: {{host.name}}\nRabbitMQ Queue: {{rabbitmq_queue.name}}",
+  "tags": [
+    "integration:rabbitmq"
+  ],
+  "options": {
+    "notify_audit": false,
+    "locked": false,
+    "timeout_h": 0,
+    "new_host_delay": 300,
+    "require_full_window": true,
+    "notify_no_data": false,
+    "renotify_interval": "0",
+    "escalation_message": "",
+    "no_data_timeframe": null,
+    "include_tags": true,
+    "thresholds": {
+      "critical": 1,
+      "critical_recovery": 0
+    },
+    "threshold_windows": {
+      "trigger_window": "last_15m",
+      "recovery_window": "last_15m"
+    }
+  },
+  "priority": null,
+  "recommended_monitor_metadata": {
+    "description": "Notify your team when a RabbitMQ queue's message unacknowledged rate is higher than usual."
```

```suggestion
    "description": "Notify your team when a RabbitMQ message unacknowledged rate is higher than usual."
```
While your version is accurate, it's a bit of a tongue twister to read, and this change still gets the point across.